suite-native/transactions/src/components/TransactionsList/TransactionListItem.tsx

```diff
@@ -18,36 +18,43 @@ import {
 import { selectFiatCurrency } from '@suite-native/module-settings';
 import { useFormatters } from '@suite-common/formatters';
 
-type AccountTransactionListItemProps = {
+import { TransactionListItemIcon } from './TransactionListItemIcon';
+
+type TransactionListItemProps = {
     transaction: WalletAccountTransaction;
 };
 
-const transactionIconMap: Partial<Record<TransactionType, IconName>> = {
-    recv: 'receive',
-    sent: 'send',
+type TransactionTypeProperties = {
+    prefix: 'From' | 'To';
+    sign: '+' | '-';
+    signColor: keyof Colors;
 };
-
-const transactionTextMap: Partial<Record<TransactionType, string>> = {
-    recv: 'Received',
-    sent: 'Sent',
+const transactionTypePropertiesMap: Partial<Record<TransactionType, TransactionTypeProperties>> = {
```

Comment:
You don't need to type this, let inheritance do it's job:

```tsx
const transactionTypePropertiesMap = {
    recv: { prefix: 'From', sign: '+', signColor: 'green' },
    sent: { prefix: 'To', sign: '-', signColor: 'red' },
} as const;
```

Will do same job in terms of typesafety because it will throw in place of usage if you for example mispell "recv" to "revc" or any of it's properties.